### PR TITLE
Create Platform Admin dashboard for root admins

### PR DESCRIPTION
**Pull request by kewitz**
Requires https://github.com/opencollective/opencollective-api/pull/10100

### CODE REVIEW

File: components/OCFBanner.tsx [hunk 1]
```diff
@@ -200,7 +199,7 @@ export function OCFBannerWithData(props: OCFBannerProps) {
   }
 
   if (query.error) {
-    return <MessageBoxGraphqlError error={query.error} />;
```
Comment (gustavlrsn) [inline components/OCFBanner.tsx:203]: What is the context for this change?

**Review by gustavlrsn — APPROVED** [review]
LGTM!

File: components/OCFBanner.tsx (hunk 1 above)
Comment (kewitz) [reply components/OCFBanner.tsx:203]: The query would throw due to the fact the `accountSlug` is invalid. I'm not sure if this is still required, I'll test it again and see what happens.

Comment (kewitz) [reply components/OCFBanner.tsx]: It is not needed anymore, it was just a development workaround. Dropped the commit.

**Pull request merged**